shoop/utils/text.py

```diff
@@ -28,8 +30,8 @@ def flatten(str, whitespace="-"):
     Flatten the given text into lowercase ASCII, removing diacriticals etc.
     Replace runs of whitespace with the given whitespace replacement.
 
-    >>> flatten(u"hellö, wörld")
-    "hello,-world"
+    >>> print(flatten("hellö, wörld"))
```

Comment:
No need to use `print` in doctests.

Just that the retval should be single-quoted (as is `repr`'s preference).